### PR TITLE
Fuller documentation of Particle class

### PR DESCRIPTION
**Pull request by eduardo-rodrigues**
Addresses https://github.com/scikit-hep/particle/issues/50.

### CODE REVIEW

**Comment by eduardo-rodrigues** [comment]
@henryiii, I will re-read it through with fresh eyes but you are welcome to review already - largely complete.

File: particle/particle/particle.py [hunk 1]
```diff
@@ -48,13 +48,102 @@ class Particle(object):
...
+    anti_flag
+        The particle-antiparticle flag.
+
+        A = B     - particle that has anti-particle partner different from particle
+                    with ASCII name formed by concatenation of the name shown below with charge
+                    ( e- <--> e+, pi+ <--> pi-, K+ <--> K-, W+ <--> W- ).
+        A = F     - particle that has anti-particle partner different from particle
+                    with ascii name formed by concatenation of the name shown below with string "bar" and charge
+                    by the rule (nu(e) <--> nubar(e), p <--> pbar, Delta++ <--> Deltabar--)
+        A = blank - particle that coincide with its antiparticle (gamma, pi0, eta).
```
Comment (henryiii) [inline particle/particle/particle.py]: Coincides

File: particle/particle/particle.py [hunk 2]
```diff
@@ -48,13 +48,102 @@ class Particle(object):
...
+        The particle name in LaTeX.
+
+    mass
+        The particle mass, in MeV.
+
+    mass_lower
+        The lower uncertainty on the particle mass, in MeV.
+
+    mass_upper
+        The lower uncertainty on the particle mass, in MeV.
```
[the remark this answers is not in the document]
Comment (eduardo-rodrigues) [reply particle/particle/particle.py]: Yep!

File: particle/particle/particle.py (hunk 1 above)
Comment (eduardo-rodrigues) [reply particle/particle/particle.py]: Thanks - I copied this info from the extended PDG file and fixed a few typos on the way. This one slipped.

**Comment by henryiii** [comment]
LGTM!

**Comment by eduardo-rodrigues** [comment]
Closes https://github.com/scikit-hep/particle/issues/50.

**Pull request merged**